### ISSUE TITLE
add sensor Beaufort based on m/s

### ISSUE THREAD

**Issue by Mariusthvdb** [briis/weatherbit#15]
HI,

since Beaufort is the scale news headlines and meteorology institutes use in our region for wind and warnings, would younplease consider adding a sensor for that too. You probably know about the scale, here's a quick link for references in this feature request:
https://en.wikipedia.org/wiki/Beaufort_scale

I use 2 template sensors myself, but it would be great if your integration could create it:

```
      windkracht:
        friendly_name: Windkracht
        value_template: >
          {% set wind = 3.6 * states('sensor.windspeed')|float %}
          {% set wind_round = wind|round(1) %}
            {% if wind <= 1 %}Bft: 0, Wind stil, {{wind_round}} km/u
            {% elif wind <= 5 %}Bft: 1, Zwak: flauwe wind, {{wind_round}} km/u
            {% elif wind <= 11 %}Bft: 2, Zwak, {{wind_round}} km/u
            {% elif wind <= 20 %}Bft: 3, Matig: lichte bries, {{wind_round}} km/u
            {% elif wind <= 28 %}Bft: 4, Matig: flauwe bries, {{wind_round}} km/u
            {% elif wind <= 39 %}Bft: 5, Vrij krachtig, {{wind_round}} km/u
            {% elif wind <= 50 %}Bft: 6, Krachtig, {{wind_round}} km/u
            {% elif wind <= 62 %}Bft: 7, Hard, {{wind_round}} km/u
            {% elif wind <= 75 %}Bft: 8, Stormachting, {{wind_round}} km/u
            {% elif wind <= 89 %}Bft: 9, Storm, {{wind_round}} km/u
            {% elif wind <= 103 %}Bft: 10, Zware storm, {{wind_round}} km/u
            {% elif wind <= 117 %}Bft: 11, Zeer zware storm, {{wind_round}} km/u
            {% else %} > 117 %}Bft: 12, Orkaan, {{wind_round}} km/u
          {%- endif %}

      wind_bft:
        friendly_name_template: >
          Bft: {{states('sensor.wind_bft')}}
        value_template: >
          {% set wind = 3.6 * states('sensor.windspeed')|float %}
            {% if wind <= 1 %}0
            {% elif wind <= 5 %}1
            {% elif wind <= 11 %}2
            {% elif wind <= 20 %}3
            {% elif wind <= 28 %}4
            {% elif wind <= 39 %}5
            {% elif wind <= 50 %}6
            {% elif wind <= 62 %}7
            {% elif wind <= 75 %}8
            {% elif wind <= 89 %}9
            {% elif wind <= 103 %}10
            {% elif wind <= 117 %}11
            {% else %} > 117 %}12
          {%- endif %}
        entity_picture_template: >
          {% set state = states('sensor.wind_bft') %}
          {% set path = '/local/weather/beaufort/' %}
          {% set ext = '.jpg'%}
            {{[path,state,ext]|join('')|lower}}

```

thanks for considering!

**Comment by briis** [comment]
Sure, I will add this. Could also be an interesting test case to do some translations. It looks like you are Dutch speaking, so I think I will do two new sensors - one with the Beaufort value and one with the text describing the value as you show above. I can steel the Dutch text from above and then default to English of no translation. Now I just need to figure out how to that.

**Comment by Mariusthvdb** [comment]
haha, almost...

no I am Dutch ;-)

oops. this makes my comment seem ridiculous. (you thought I was German before the edit ...)

**Comment by briis** [comment]
Yes I did - but then I read your text more carefully. I am actually the VP for an IT company managing companies across Europe, and Holland is one of them - so I should have spotted it right away. Anyway, you saw I changed it.

**Comment by briis** [comment]
Fixed in release 0.22
